include/uv.h

```diff
@@ -345,11 +345,32 @@ typedef void (*uv_random_cb)(uv_random_t* req,
                              void* buf,
                              size_t buflen);
 
+typedef enum {
+  UV_CLOCK_MONOTONIC,
+  UV_CLOCK_REALTIME
+} uv_clock_id;
```

Comment:
What are the considerations for (not) including `CLOCK_BOOTTIME`?

Reply:
Not every platform has an (easy/reliable/&c) way of obtaining it. I may add it in the future.